File: 프로그래머스/lv0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        if(n % 7 == 0) {
            // 7로 나누어 떨어진다
            answer = n / 7;
        } else {
            // 안떨어진다
            answer = n / 7 + 1;
        }
        
        return answer;
    }
}

File: 프로그래머스/lv0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.js
function solution(n) {
    // 1 ~ 7명 = 0.XX ~ 1
    // 8 ~ 14명 = 1.XX ~ 2
    // 15 ~ 21명 = 2.XX ~ 3
    // n을 7로 나눈 값을 올림처리
    var answer = Math.ceil(n / 7);
    return answer;
}